modules/virt-about-cpu-and-memory-quota-namespace.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
+
+[id="virt-about-cpu-and-memory-quota-namespace_{context}"]
+= About CPU and memory quotas in a namespace
+
+A resource quota, defined by the `ResourceQuota` object imposes restrictions on
+a namespace that limit the total amount of compute resources that can be
```

Comment:
I think this could be rephrased to be more customer-focused and minimalistic. 

Suggestion: You can use a _resource quota_ to limit the consumption of compute resources within a namespace. Define a `ResourceQuota` object to set restrictions on [insert specific types of resources here].

Reply:
In the context of this module, the compute resource quota is already defined with default values. So, maybe it is out of scope to provide a more detailed explanation on the types of resources that you can configure? 

I could add a .Additional Resources section and link to this assembly: https://docs.openshift.com/container-platform/4.4/applications/quotas/quotas-setting-per-project.html. Let me know what you think.

Reply:
> @ousleyp In the context of this module, the compute resource quota is already defined with default values in place. So, maybe it is out of scope to provide a more detailed explanation on the types of resources that you can configure with the `ResourceQuota` object?

Understood! 

> 
> I could add a `.Additional Resources` section and link to this assembly in the OpenShift docs: https://docs.openshift.com/container-platform/4.4/applications/quotas/quotas-setting-per-project.html.
> 
> Let me know what you think.

This sounds like a good idea to me. I'm glad you found that! Let me know if you have any trouble with the xref.